packages/lib/CalendarService.ts

```diff
@@ -337,7 +337,7 @@ export default abstract class BaseCalendarService implements Calendar {
     const userTimeZone = userId ? await this.getUserTimezoneFromDB(userId) : "Europe/London";
     const events: { start: string; end: string }[] = [];
     objects.forEach((object) => {
-      if (object.data == null || JSON.stringify(object.data) == "{}") return;
+      if (!object || object.data == null || JSON.stringify(object.data) == "{}") return;
```

Comment:
Checks if object exists before trying to access properties within.

Reply:
Proper types should've prevented this.

Reply:
Agreed, we'd have had to type these ourselves and that would have been error prone too - types do not prevent this.